packages/create-ripple-app/template/_.env

```diff
@@ -20,12 +20,10 @@ SEARCH_AUTH_USERNAME=<%= searchusername %>
 SEARCH_AUTH_PASSWORD=<%= searchpassword %>
 
 # Disable basic auth in local env.
-BASIC_AUTH=1
+BASIC_AUTH=0
```

Comment:
This need to be 1 as lagoon still use it for PR env, unless we have playbook setup for new project env without commit .env file.

Reply:
fixed